app/assets/stylesheets/application.css.scss

```diff
@@ -6,3 +6,16 @@
 @import "daplog/base";
 @import "daplog/blocks";
 @import "daplog/contexts";
+
+#foo{}
+a.Bar   {
+    margin : 0 0 0;
+	backgrounds:url(https://example.com/assets/image.png) ;
+  & > { color: red; }
+  content: 'foo'! important;
+  -webkit-border-radius: 0px .500em;
+  border: none solid rgba(0,0,0,0); /* baz */
+  @extend #foo;
+
+  border: 1.0px  solid #ffffFF
```

Comment:
Property `border` already defined on line 17<br>Color `#ffffFF` should be written as `#ffF`<br>Color `#ffffFF` should be written as `#ffffff`<br>Declaration should be terminated by a semicolon<br>`1.0` should be written without a trailing zero as `1`<br>`1.0px` should be written without the mantissa as `1px`